src/pages/bp-role-categories/forms/RoleCategoriesForm.js

```diff
@@ -0,0 +1,148 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect, useState } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { FormControlLabel, Checkbox } from '@mui/material'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { BusinessPartnerRepository } from 'src/repositories/BusinessPartnerRepository'
+import { useForm } from 'src/hooks/form'
+import { ControlContext } from 'src/providers/ControlContext'
+
+export default function RoleCategoriesForm({ labels, maxAccess, recordId }) {
+  const [editMode, setEditMode] = useState(!!recordId)
+
```

Comment:
use 
const editMode = !!formik.values.recordId || !!recordId
instead of the useState please

Reply:
done